src/dbnode/storage/series/buffer.go

```diff
@@ -977,7 +980,7 @@ type BufferBucketVersions struct {
 	buckets           []*BufferBucket
 	start             time.Time
 	opts              Options
-	lastReadUnixNanos int64
+	lastReadUnixNanos *atomic.Int64
```

Comment:
don't see you allocating using the atomic.NewInt64 call, is the nil value usable for this type?